### PR TITLE
Feature/cards page ui

### PR DESCRIPTION
**Pull request by delmontemica**
FIXED card flip transition
ADDED progress per card

### CODE REVIEW

File: frontend_vue/src/assets/scss/style.scss [hunk 1]
```diff
@@ -130,4 +130,82 @@ div{
...
+  &-options{
+    margin-left: 40px;
+    position: relative;
+    display: inline-block;
+  }
+  &-content{
+    text-align: center;
+    line-height: 26px;
+    font-size: 20px;
+    padding: 150px 150px;
```
Comment (hugh-n) [inline frontend_vue/src/assets/scss/style.scss]: Change to `padding: 150px;`

File: frontend_vue/src/components/Card.vue [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+            </v-flex>
+          </v-layout>
+        </v-container>
+    </v-content>
+</template>
+
+<script>
+    export default {
+        data: () => ({
+          flipped: false,
```
Comment (hugh-n) [inline frontend_vue/src/components/Card.vue]: Previous line is 4 space indent but this line is 2 space indent. Please, make all your indents the same number of spaces.

File: frontend_vue/src/assets/scss/style.scss (hunk 1 above)
Comment (delmontemica) [reply frontend_vue/src/assets/scss/style.scss]: Fixed.

File: frontend_vue/src/components/Card.vue (hunk 2 above)
Comment (delmontemica) [reply frontend_vue/src/components/Card.vue]: Fixed

**Pull request merged**